optifit backend/pushup_counter.py

```
@@ -0,0 +1,353 @@
+        hip_center = [(left_hip.x + right_hip.x) / 2, (left_hip.y + right_hip.y) / 2]
+        ankle_center = [(left_ankle.x + right_ankle.x) / 2, (left_ankle.y + right_ankle.y) / 2]
+        
+        # Calculate alignment angles
+        shoulder_hip_angle = np.arctan2(hip_center[1] - shoulder_center[1], hip_center[0] - shoulder_center[0])
+        hip_ankle_angle = np.arctan2(ankle_center[1] - hip_center[1], ankle_center[0] - hip_center[0])
+        
+        # Calculate alignment deviation (perfect alignment = small deviation)
+        angle_diff = abs(shoulder_hip_angle - hip_ankle_angle)
+        alignment_score = max(0, 100 - (angle_diff * 180 / np.pi) * 10)
+        
+        return alignment_score
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)